### ISSUE TITLE
Occasionally there will be a failure

### ISSUE THREAD

**Issue by liby** [ChrisAnd1998/FalconX#19]
### OS
  -  Win10 17763.678

### Problem
  1. After the computer restarts,Or is it waken up from the sleep state of the computer
  2. Or is it waken up from the sleep state of the computer,Restart the software to solve the problem
  3. Incidental incident, no law is found for the time being

###  ScreenShot
![demo](https://user-images.githubusercontent.com/38807139/63148957-4da57700-c035-11e9-95ca-b36dea6e5c38.gif)

**Comment by user00101010** [comment]
Not Chris, but can confirm this isn't happening on my system. I can see that you are using another third-party app to display the network activity and CPU usage in the taskbar -- out of curiosity what's the name of the app, it looks quite nice? Also, could you please test without said application, to see if it's interfering with FalconX

**Comment by liby** [comment]
@user00101010 
I don't know if it counts as advertising. If it does, I can delete it
[TrafficMonitor](https://github.com/zhongyang219/TrafficMonitor)

**Comment by ChrisAnd1998** [comment]
Seems like your taskbar's Redraw gets enabled. Causing the taskbar to update and go to it's origin position. TrafficMonitor is monitoring your traffic "of course" and outputs it to it's own toolbar in the taskbar causing the taskbar to update (Enabling the taskbar's Redraw).

**Comment by liby** [comment]
- Maybe it’s not clear enough for me to describe
  - This issue has little to do with TrafficMonitor, and it doesn't even matter.
- Software failure only because the resource manager is restarted
- Here is a bat code that can restart the resource manager, you can try
```bat
@echo off
taskkill /f /im explorer.exe
CD /d %userprofile%\AppData\Local
DEL IconCache.db /a
start explorer.exe
echo END
```
![demo](https://user-images.githubusercontent.com/38807139/65201134-1571d800-dabc-11e9-8070-63c0872c15d0.gif)

**Comment by ChrisAnd1998** [comment]
Does it also happen without restarting Explorer.exe? 

If Explorer.exe gets killed or restarted then FalconX loses it's handle's and won't be able to move it until you restart FalconX. 

I will test your script as soon as i have more time too. And will try to fix it...

**Comment by ChrisAnd1998** [comment]
As you can see now in the new update 1.1.7.0 you can see it restarts automatically when explorer.exe got killed.

**Comment by liby** [comment]
- Test can, thank you
- Give a suggestion, the interface can stretch, It’s better if the content can fill the interface.
- In the picture, after the interface is stretched, there will be a large blank.
![image](https://user-images.githubusercontent.com/38807139/65373658-ebced180-dcb2-11e9-861e-a7bbd03eabf9.png),

**Comment by ChrisAnd1998** [comment]
Allright :)

Will be fixed in 1.1.8.0

Here is the Pre-Release
https://chrisandriessen.nl/web/downloads/falconx/Pre-Release/Pre-Release%201.1.8.0.zip

**Comment by liby** [comment]
- hum....
- In fact, these are optimization items.
- There's no need for you to rush
- You can optimize slowly
This software is great.

![Snipaste_2019-09-21_21-58-50](https://user-images.githubusercontent.com/38807139/65374316-394f3c80-dcbb-11e9-92f3-276c5a50b182.png)
